app/components/views/Home/index.tsx

```diff
@@ -65,6 +69,19 @@ const useWeb3Modal = () => {
             darkMode: false, // Optional. Use dark theme, defaults to false
           },
         },
+        torus: {
+          package: Torus,
+          options: {
+            networkParams: {
+              host: "matic", // optional
+            },
+          },
+          display: {
+            name: "Email, Google, Twitter",
+            description: "Use your existing services to connect.",
```

Comment:
oh neat, so in the future we can use these overrides to provide translations! I'll create a ticket for that